Jumpscale/builder/db/BuilderEtcd.py

```diff
@@ -44,17 +44,39 @@ def client_get(self, name):
         """
         return j.clients.etcd.get(name)
 
-    def test(self):
+    def start(self, config_file=None):
+        """start etcd
+
+        :param config_file: config file path, defaults to None
+        :type config_file: str, optional
+        :return: tmux pane
+        :rtype: tmux.Pane
         """
-        js_shell 'j.builder._template.test()'
-        :return:
+        cmd = self.tools.joinpaths(j.core.dirs.BINDIR, self.NAME)
+        if config_file:
+            cmd += ' --config-file %s' % config_file
+        return j.tools.tmux.execute(cmd, window=self.NAME, pane=self.NAME, reset=True)
+
+    def stop(self, pid=None, sig=None):
+        """Stops traefik process
+
+        :param pid: pid of the process, if not given, will kill by name
+        :type pid: long, defaults to None
+        :param sig: signal, if not given, SIGKILL will be used
+        :type sig: int, defaults to None
+        """
+        if pid:
+            j.sal.process.kill(pid, sig)
+        else:
+            j.sal.process.killProcessByName(self.NAME, sig)
```

Comment:
if you add the bin dir, you'll make sure you're killing the right process. This might match on anything that contains `etcd` in the name.